packages/http/src/util.js

```diff
@@ -196,3 +196,5 @@ export {
    */
   uuid,
 };
+
+export { field } from '@openfn/language-common/util';
```

Comment:
This is better now: we can just export what we want out of common util. Docs will automatically be linked up